### ISSUE TITLE
Autofill on Synology weblogin doesn't work anymore

### ISSUE THREAD

**Issue by Timo-github** [pfn/passifox#494]
I have a Synology NAS (DS213j). Since the update of its operating system DSM to version 6.0 the login credentials on the weblogin page aren't filled in automatically anymore. On other pages it still works fine.

I work on a windows pc with:
- ChromeIpass on Chrome (lastest version)
- Keepasshttp (1.8.4.1)
- Keepass (2.32)
- URL 1 (not working): http://www.[personal URL].nl:5000/
- URL 2 (not working): [external IP-adres]:5000/

I tried both URL's where I added the URL in the exact way into the URL-field in my Keepass entry (same as i do for other pages).

**What can I do? Does anybody have an answer?**

**Comment by derandiunddasbo** [comment]
Sorry, no answer. I have the same problem and wonder, if we are the only two humans on earth using cromeIPass / PassIFox together with a Synology DSM 6, as even a google gives exactly one search result mentioning this particular problem: your post.

At least there is a workaround (if you didn't know it already): Right clicking on the user name field opens the Chrome context menu and chosing 'chromeIPass > Fill User + Pass' fills in the credentials at least semi-automatically.

Unfortunately, choosing "Choose own credential fields for this page" makes things even worse. It finds the username and password fields on the page correctly. But on a page reload the credentials are still not filled in. And from then on even the semi-automatic fill in from the context menu is broken, because the password fields isn't filled anymore. For fixing this, the page specific setting for the DSM login page has to be removed again from the "Specified credential fields" list in chromeIPass settings.

Bottom line: I'd also like to see this bug fixed. In addition, as I use multiple DSM logins, I have to close the "More than one Logins found in KeePass" warning every time I use 'Fill User + Pass', as the chromeIPass dropdown for selecting credentials only works after using the 'Fill User + Pass' menu.

**Comment by Timo-github** [comment]
Thanks for your update. I didn't know the workaround yet. Indeed, it doesn't solve the problem but helps a little to save time (at least for me with only 1 Synology login page).

But it's also interesting that only we  seem to be struggling with that autofill-problem...

I'm curious if we receive more reactions....

**Comment by derandiunddasbo** [comment]
Just in case, someone stumbles upon this while searching for solutions for this issue: Recently I gave up waiting for this issue to be resolved or even investigated.

I found a really good alternative: The "Kee" plugin for chrome. 

Not only does it fill the Synology DSM Login without issues, it also has a better (i.e. more advanced) configuration, f.e. configurable RegEx based URL detection and configurable credentials field detection.

The migration was easy. The plugin worked out of the box for most sites and only needed some config tweaking for showing only relevant credentials for some sites with multiple logins or multiple URLs.

**Comment by KyleS-ITW** [comment]
Thanks for posting this update, i have been having this problem the last 2 years and periodically look for options to fix it and using Kee as you suggested is working perfectly.

**Comment by ghost** [comment]
Synology has a fix for this now apparently.  Part of the 'Version: 6.2.2-24922-2' release.

https://www.synology.com/en-ca/releaseNote/DS918+#ver_24922-2
